src/learn/augur-to-auspice.rst

```diff
@@ -206,6 +213,100 @@ If a coloring was provided for this key then Auspice will attempt to display
 the title rather than the key. Selecting filters here behaves the same was as
 sidebar data filtering (:ref:`see [a2], above <auspice-component-sidebar-filter>`).
 
+.. _auspice-component-diversity-panel:
+
+Diversity (Entropy) Panel
+---------------------------------
+
+We'll use the following annotated screenshot of the diversity (entropy) panel
+of the  `measles virus tree <https://nextstrain.org/measles>`__ (built using
+`this Augur pipeline <https://github.com/nextstrain/measles>`__) to introduce the
+various components:
+
+.. image:: ../images/auspice-components-diversity-panel.png
+  :alt: Annotated screenshot of Auspice's diversity (entropy) panel
+
+The diversity panel is enabled by data in the :term:`dataset JSON <dataset>`.
+The top-level ``meta.genome_annotations`` provides the genome annotations
+displayed and the individual tree nodes provide the mutations
+via ``node.branch_attrs.mutations``, which are used to calculate the entropy
+and to count the mutation events.
+
+Gaps (``-``), masked nucleotides (``N``), and unknown amino acids (``X``) are
```

Comment:
Looking at the code, gaps are only considered for nucleotides (not AA)

Reply:
Ah, thanks! Yup, both [calcMutationCounts](https://github.com/nextstrain/auspice/blob/0b5163aaee46cc14c6ef51f070c25f46e330c635/src/util/entropy.js#L33-L52) and [calcEntropy](https://github.com/nextstrain/auspice/blob/0b5163aaee46cc14c6ef51f070c25f46e330c635/src/util/entropy.js#L112-L114) only look for `-` in nucleotides.